polus-csv-statistics-plugin/src/main.py

```diff
@@ -118,6 +137,26 @@ def kurtosis(data_list,data_dict):
     parser.add_argument('--outDir', dest='outDir', type=str,
                         help='Output collection', required=True)
     
+    # Parse the arguments
+    args = parser.parse_args()
+    statistics = args.statistics.split(',')
+    logger.info('statistics = {}'.format(statistics))
+    inpDir = args.inpDir    # Initialize the logger
+    logging.basicConfig(format='%(asctime)s - %(name)-8s - %(levelname)-8s - %(message)s',
+                        datefmt='%d-%b-%y %H:%M:%S')
+    logger = logging.getLogger("main")
+    logger.setLevel(logging.INFO)
+
+    # Setup the argument parsing
+    logger.info("Parsing arguments...")
+    parser = argparse.ArgumentParser(prog='main', description='Calculate simple statistics to groups of data in a csv file.')
+    parser.add_argument('--statistics', dest='statistics', type=str,
+                        help='Types of statistics to calculate', required=True)
+    parser.add_argument('--inpDir', dest='inpDir', type=str,
+                        help='Input csv collection to be processed by this plugin', required=True)
+    parser.add_argument('--outDir', dest='outDir', type=str,
+                        help='Output collection', required=True)
+
```

Comment:
I'm not sure what this section is doing here. It looks like a duplicate of the above code.